components/dashboard/src/App.tsx

```diff
@@ -37,7 +37,6 @@ const Projects = React.lazy(() => import(/* webpackPrefetch: true */ './projects
 const Project = React.lazy(() => import(/* webpackPrefetch: true */ './projects/Project'));
```

Comment:
**suggestion**: Adding a confirmation modal for deleting a project could be really helpful and less surprising to the user! Feel free to add a follow up issue for this if needed. 🏓 

> **Title**: Remove Project
> **Paragraph**: Are you sure you want to remove this project from this team? Team members will also loose access to this project.
> **Actions**: Cancel, Remove Project

Reply:
let's tackle this separately.

Reply:
DEAL 🤝